doc/locale/en/api_reference.pot

```diff
@@ -163,7 +166,10 @@ msgstr ""
 msgid "Value to put in the data stream."
 msgstr ""
 
-msgid "Returns a concatenation of ``counter_obj:collect()`` across all internal counters of ``summary_obj``. For ``observation`` description, see :ref:`counter_obj:collect() <counter-collect>`."
+msgid "Table containing label names as keys, label values as values (table). A new value is observed by all internal counters with these labels specified. Label ``\"quantile\"`` are not allowed in ``summary``. It will be added automatically. If ``max_age_time`` and ``age_buckets_count`` are set, observed value will be added to each bucket."
```

Comment:
```suggestion
msgid "A table containing label names as keys, label values as values (table). A new value is observed by all internal counters with these labels specified. Label ``\"quantile\"`` are not allowed in ``summary``. It will be added automatically. If ``max_age_time`` and ``age_buckets_count`` are set, the observed value will be added to each bucket."
```

Reply:
It's autogenerated doc, I fixed it in original file